manifests/metallb-frr.yaml

```diff
@@ -577,6 +577,7 @@ rules:
   - services
   - endpoints
   - nodes
+  - secrets
```

Comment:
I tried with having secrets in Role, but below error occurred from reconciler while listing secrets from metallb-system namespace, so still holding it in cluster role. 
`E0317 23:21:27.475103     100 reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.2/tools/cache/reflector.go:167: Failed to watch *v1.Secret: failed to list *v1.Secret: secrets is forbidden: User "system:serviceaccount:metallb-system:speaker" cannot list resource "secrets" in API group "" at the cluster scope`
Any idea what can be done to avoid this ?